src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -146,144 +149,181 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
+                {hasStorageBuckets && (
+                  <CheckboxInput
+                    id="features_storage_buckets"
+                    name="features_storage_buckets"
+                    label="Storage buckets"
+                    onChange={() => {
+                      ensureEditMode(formik);
+                      void formik.setFieldValue(
+                        "features_storage_buckets",
+                        !formik.values.features_storage_buckets,
+                      );
+                    }}
+                    checked={formik.values.features_storage_buckets}
+                    disabled={isDefaultProject || isNonEmpty}
+                  />
+                )}
                 <CheckboxInput
-                  id="features_storage_buckets"
-                  name="features_storage_buckets"
-                  label="Storage buckets"
+                  id="features_storage_volumes"
+                  name="features_storage_volumes"
+                  label="Storage volumes"
                   onChange={() => {
                     ensureEditMode(formik);
                     void formik.setFieldValue(
-                      "features_storage_buckets",
-                      !formik.values.features_storage_buckets,
+                      "features_storage_volumes",
+                      !formik.values.features_storage_volumes,
                     );
                   }}
-                  checked={formik.values.features_storage_buckets}
+                  checked={formik.values.features_storage_volumes}
                   disabled={isDefaultProject || isNonEmpty}
                 />
-              )}
-              <CheckboxInput
-                id="features_storage_volumes"
-                name="features_storage_volumes"
-                label="Storage volumes"
-                onChange={() => {
-                  ensureEditMode(formik);
-                  void formik.setFieldValue(
-                    "features_storage_volumes",
-                    !formik.values.features_storage_volumes,
-                  );
-                }}
-                checked={formik.values.features_storage_volumes}
-                disabled={isDefaultProject || isNonEmpty}
-              />
-            </>
-          )}
+              </>
+            )}
+          </div>
+
           <hr />
+          <StoragePoolSelector
+            value={formik.values.default_instance_storage_pool}
+            setValue={(value) =>
+              void formik.setFieldValue("default_instance_storage_pool", value)
+            }
+            selectProps={{
+              label: "Default instance storage pool",
+              disabled:
+                (formik.values.features_profiles === false &&
+                  features === "customised") ||
+                isEdit,
+              help: isEdit ? (
+                <>
+                  Edit the storage pool in the{" "}
+                  <ResourceLink
+                    type="instance"
```

Comment:
```suggestion
                    type="profile"
```
We should use the correct type, this is a profile, not an instance.